### PR TITLE
Make a team

### PR DESCRIPTION
**Pull request by elreimundo**

### CODE REVIEW

File: app/views/team.erb [hunk 1]
```diff
@@ -1,19 +1,21 @@
-<%= @queried_team.name %>
+<h3><%= @queried_team.name %></h3>
 <div id="team_members">
   <% unless @queried_team.politicians.nil? %>
-<% @queried_team.politicians.each do |sleazeball| %>
-<p><%= sleazeball.info %></p>
-<% end %>
-<% end %>
+    <% @queried_team.politicians.each do |sleazeball| %>
```
Comment (nscricco) [inline app/views/team.erb:4]: :+1: making fun of politicians

File: app/controllers/index.rb [hunk 2]
```diff
@@ -15,9 +15,19 @@
   json @queried_team
 end
 
+get '/teams/create' do
+  redirect "/teams/#{current_user.teams.first.id.to_s}" unless current_user.teams.empty?
```
Comment (nscricco) [inline app/controllers/index.rb:19]: code smell spotted.  no worries for now

File: app/controllers/index.rb [hunk 3]
```diff
@@ -15,9 +15,19 @@
...
+  erb :create_team
+end
```
Comment (zspencer) [inline app/controllers/index.rb:21]: This seems to be enforcing that a user can have only one team. What if the function was `redirect "/teams/#{current_users.active_team.id}" if current_user.has_active_team?`

**Comment by nscricco** [comment]
Looks good boys.  No changes necessary.

File: app/controllers/index.rb (hunk 3 above)
Comment (zspencer) [inline app/controllers/index.rb:21]: REVEAL INTENNNTTTTTT

**Pull request merged**